source/base/Event.ooc

```diff
@@ -17,14 +17,20 @@
 
 Event: class {
 	_head: Func
-	head ::= this _head
 	_tail: This
+	head ::= this _head
```

Comment:
You may be right. We seem to be splitting them up a lot more here than in the other project though, so there will be more to change there.